api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
@artstorm i think it is better practice to make the check `if ($e instanceof HttpExceptionInterface)` if you want to check for the presence of the `getResponse` method. As the `ValidationException` needs the `getResponse` method *and* the `getStatusCode` method of `HttpExceptionInterface` I think the `ValidationException` should be defined as 
```php
ValidationException extends HttpResponseException implements HttpExceptionInterface
```
so that both methods are available.